src/printbox-ext-plot/PrintBox_ext_plot.mli

```diff
@@ -0,0 +1,61 @@
+(* This file is free software. See file "license" for more details. *)
+
+(** {1 Extend {!PrintBox.t} with plots of scatter graphs and line graphs} *)
+
+type plot_spec =
+  | Scatterplot of {
+      points: (float * float) array;
```

Comment:
it'd be good to have docstrings on the fields to explain their purpose. For, say, `Boundary_map` it's a bit less clear than for `Scatterplot`.

Reply:
Done in next commit.